client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,214 @@
+		canUserUseCustomerHome: PropTypes.bool.isRequired,
+	};
+
+	render() {
+		const { translate, canUserUseCustomerHome } = this.props;
+
+		if ( ! canUserUseCustomerHome ) {
+			return (
+				<EmptyContent
+					title={ preventWidows(
+						translate( 'Only site administrators can view the Customer Home.' )
+					) }
+					line={ preventWidows(
+						translate(
+							"Ensure you have selected the right site on the sidebar. If it's correct, you might want to contact the administrator of this site."
+						)
+					) }
+					illustration="/calypso/images/illustrations/error.svg"
+				/>
+			);
+		}
+
+		const { customizeUrl, site, siteSlug } = this.props;
+
+		return (
+			<Main className="customer-home__main is-wide-layout">
+				<PageViewTracker path={ `/home/:site` } title={ translate( 'Customer Home' ) } />
+				<DocumentHead title={ translate( 'Customer Home' ) } />
+				<SidebarNavigation />
+				<div className="customer-home__layout">
+					<div className="customer-home__layout-col">
+						<Card>
+							<CardHeading>{ translate( 'My Site' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Review and update my site' ) }
+							</h6>
+							<div className="customer-home__card-button-pair">
+								<Button href={ site.URL } primary>
+									{ translate( 'View Site' ) }
+								</Button>
+								<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+							</div>
+						</Card>
+						<Card className="customer-home__card-boxes">
+							<div className="customer-home__boxes">
+								<ActionBox
+									action={ () => page( `/page/${ siteSlug }` ) }
+									label={ translate( 'Add a page' ) }
+									iconSrc="/calypso/images/customer-home/page.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/post/${ siteSlug }` ) }
+									label={ translate( 'Write blog post' ) }
+									iconSrc="/calypso/images/customer-home/post.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Customize theme' ) }
+									iconSrc="/calypso/images/customer-home/customize.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/themes/${ siteSlug }` ) }
+									label={ translate( 'Change theme' ) }
+									iconSrc="/calypso/images/customer-home/theme.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Edit menus' ) }
+									iconSrc="/calypso/images/customer-home/menus.svg"
+								/>
+								<ActionBox
+									action="https://en.support.wordpress.com/images/"
+									label={ translate( 'Change images' ) }
+									iconSrc="/calypso/images/customer-home/images.svg"
+								/>
+								<ActionBox
+									action="https://logojoy.grsm.io/looka"
+									label={ translate( 'Design a logo' ) }
+									iconSrc="/calypso/images/customer-home/logo.svg"
+								/>
+								<ActionBox
+									action="https://wordpress.com/email/domain.wordpress.com"
```

Comment:
I'm assuming this is a placeholder which needs updating. :)

Reply:
Thanks, good catch, fixed 👍